src/copilot/inspect/InspectionCopilot.ts

```diff
@@ -132,43 +137,53 @@ export default class InspectionCopilot extends Copilot {
     }
 
     public async inspectRange(document: TextDocument, range: Range | Selection): Promise<Inspection[]> {
-        // ajust the range to the minimal container class or (multiple) method symbols
-        const methods: SymbolNode[] = await getIntersectionMethodsOfRange(range, document);
-        const classes: SymbolNode[] = await getClassesContainedInRange(range, document);
-        const symbols: SymbolNode[] = [...classes, ...methods];
-        if (symbols.length < 1) {
-            const containingClass: SymbolNode = await getInnermostClassContainsRange(range, document);
-            symbols.push(containingClass);
+        if (this.busy) {
```

Comment:
this.busy()?

Reply:
`get busy()` is a getter, no need to add `()`